.python-version

```diff
@@ -0,0 +1 @@
+3.7.0
```

Comment:
Tests on CI are still using python 3.6

Reply:
Good catch @juanpaucar , is updated now.